eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver.php

```diff
@@ -265,14 +265,15 @@ private function warnAboutTooEarlyLoadedParams()
         }
 
         $logger = $this->container->get('logger');
-        foreach ($this->tooEarlyLoadedList as $param => $services) {
+        foreach ($this->tooEarlyLoadedList as $blame => $params) {
             $logger->warning(sprintf(
-                'ConfigResolver was used to load parameter "%s" before SiteAccess was loaded by services: %s. This can cause issues. '
-                . 'Try to use ConfigResolver lazily, '
+                'ConfigResolver was used by "%s" before SiteAccess was initialized, loading parameter(s) '
+                . '%s. As this can cause very hard to debug issues, please '
+                . 'try to use ConfigResolver lazily, '
                 . (PHP_SAPI === 'cli' ? 'make commands that rely on them lazy, ' : '')
-                . 'or try to mark the service as lazy.',
-                $param,
-                '"' . implode($services, '", "') . '"'
+                . 'make the service lazy or see if you can inject another one which already is.',
```

Comment:
```suggestion
                . 'make the service lazy or see if you can inject another lazy service.',
```
Not sure if that's exactly the intention, but this is more understandable